core/src/main/kotlin/behavior/GuildScheduledEventBehavior.kt

```diff
@@ -27,18 +23,32 @@ import kotlin.contracts.contract
 public interface GuildScheduledEventBehavior : KordEntity, Strategizable {
     public val guildId: Snowflake
 
+
     /**
      * Requests all the users which are interested in this event.
      *
      * @throws RequestException if anything goes wrong during the request
      */
-    public val users: Flow<User> get() = flow {
-        kord.rest.guild.getScheduledEventUsers(guildId, id).users.forEach {
-            val userData = UserData.from(it)
-            emit(User(userData, kord, supplier))
-        }
-    }
 
+    public val users: Flow<User>
+        get() = getGuildScheduledEventUsersAfter()
```

Comment:
```suggestion
    public val users: Flow<User>
        get() = supplier.getGuildScheduledEventUsers(guildId, id)
```
this is what I mean, should have used the suggestion feature before but had to find it first